src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,72 @@
+@import "../utils/variables";
+
+.stop-watch {
+  position: relative;
+  height: $watch-size;
+  width: $watch-size;
+  border: $arrow-width dashed #ff2201;
+  border-radius: 50%;
+  background-color: #fff;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    height: $center-size;
+    width: $center-size;
+    border-radius: 50%;
+    background-color: #ff2201;
+
+  }
+
+  &__minute-arrow {
+    position: absolute;
+    top: $top-minute;
+    left: $left;
+    height: $heigth-minute;
+    width: $arrow-width;
+    background-color: #ff2201;
+    transform-origin: bottom center;
+    animation: minute-arrow 3600s linear infinite;
+  }
+
+  @keyframes minute-arrow {
+    0% {
+      transform: rotate(0deg);
+    }
+
+    50% {
+      transform: rotate(180deg);
+    }
+
+    100% {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
Code for minute-arrow and second-arrow repeated. Use the DRY principle. Rename and use once animation.